### PR TITLE
Bloodsucker buffs (again)

### PR DESCRIPTION
**Pull request by Sniblet**
# Document the changes in your pull request

This antag is SO SAD.

- Blood altars are less demanding early on and candelabra are less demanding, candelabra removes mindshields in 20 seconds instead of 60 seconds.
- Frenzy now removes damage slowdown instead of reducing(?) it (I don't know if that even worked).
- Toreador don't get a humanity hit for trying to bite rats, because bloodsuckers can't even try to eat rats anyway because code.
- Lunge is instant at rank 2 instead of 3, mesmerize mutes at rank 1 instead of 2 (abilities start at rank 0).
- Changelings, cultists, heretics, and obsessed(?!) can be vassalized, because we don't have dynamic every round and them meeting should be as cool as it sounds.
- Possessed armor are now in the bloodsucker mob faction
- Lasombra are now also in the bloodsucker mob faction, leaving only Toreador and clanless (too human lol play a real clan)

# Wiki Documentation

- Lunge is instant at rank 2 instead of 3, mesmerize mutes at rank 1 instead of 2
- Spellcheck the wiki page, you spelled subservient wrong

# Changelog

:cl:   
Tons of bloodsucker tweaks:
tweak: Blood altars require less blood and fewer hearts before rank 8
tweak: Candelabra costs 50 blood from 150 to start de-shielding, takes 20 seconds to finish from 60
tweak: Frenzy removes damage slowdown instead of decreasing it
tweak: Predatory Lunge is instant at rank 2 instead of 3, Mesmerize mutes at 1 instead of 2
tweak: Changelings, cultists, heretics, and obsessed can be vassalized
tweak: Possessed armor will only attack Toreador or clanless bloodsuckers (too human play a real clan)
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
this antag is bad because the core mechanics are bad and it cannot be fixed with number changing

**Comment by Moltijoe** [comment]
> * Frenzy now removes damage slowdown instead of reducing(?) it (I don't know if that even worked).

it worked, reducing damage slowdown isn't what you'd think though, it just let you get lower health before applying the slowdown rather than reducing the slowdown itself

**Comment by Sniblet** [comment]
> this antag is bad because the core mechanics are bad and it cannot be fixed with number changing

It can be improved

**Comment by SomeguyManperson** [comment]
> > * Frenzy now removes damage slowdown instead of reducing(?) it (I don't know if that even worked).
> 
> it worked, reducing damage slowdown isn't what you'd think though, it just let you get lower health before applying the slowdown rather than reducing the slowdown itself

This may not have been noticable as blood is used to determine nutriment and anyone with mood disabled would therefore be getting hunger slowdown while in frenzy

**Comment by Sniblet** [comment]
Under VtM lore I can't think of any reason why blood should dictate nutriment, it's not an actual need so much as an addiction
Every day I ask myself what clowns coded this antag

**Comment by chemistrymain2** [comment]
> > this antag is bad because the core mechanics are bad and it cannot be fixed with number changing
> 
> It can be improved

true

i personally feel like bloodsucker has to go the way of the dinosaur-heretic does everything it does but better-but it might as well be made not dogshit

File: code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	desc = "Spring at a humanoid to grapple them without warning, or tear the dead's heart out. Attacks from concealment or the rear may even knock them down if strong enough."
 	button_icon_state = "power_lunge"
 	power_explanation = "<b>Predatory Lunge</b>:\n\
-		Click any player to instantly dash at them if above power level 3, aggressively grabbing them.\n\
-		If below level 3, you will have to charge your lunge for a while. During this time you'll have to stand still for lunge to work\n\
+		Click any player to instantly dash at them if above power level 1, aggressively grabbing them.\n\
+		If below level 2, you will have to charge your lunge for a while. During this time you'll have to stand still for lunge to work.\n\
```
Comment (Moltijoe) [inline code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm:7]: should this say 2 both times?

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm (hunk 1 above)
Comment (Sniblet) [reply code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm:7]: .>1 = 2, <2 =/ 2
It shouldn’t

Comment (Moltijoe) [reply code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm:7]: could it be rephrased so both numbers are the same then
if i mixed it up at a glance, others certainly will too

Comment (Sniblet) [reply code/modules/antagonists/bloodsuckers/powers/targeted/lunge.dm:7]: tried to do it lazily with as little change as possible, not registering that the gamemode's typical godawful communication skills would make me look the fool

**Comment by Sniblet** [comment]
Oh what do you mean it's been DNC for 4 days
Well if Github doesn't tell me the problem in language I understand I'll probably just be removing the mute on bite

**Comment by Sniblet** [comment]
I concede
It is no longer in this PR
I'll think about aggressive bite muting some other day, and it'll get nuked for making bloodsuckers OP/not modular/some other thing I can't understand

**Comment by Moltijoe** [comment]
> I concede It is no longer in this PR I'll think about aggressive bite muting some other day, and it'll get nuked for making bloodsuckers OP/not modular/some other thing I can't understand

ah whoops, musta missed that
just change it to
ADD_TRAIT(target, TRAIT_MUTE, BLOODSUCKER_TRAIT);
and
REMOVE_TRAIT(target, TRAIT_MUTE, BLOODSUCKER_TRAIT);

target. isn't required because the proc is a universal proc that gets the target from it's parameters

**Review by Moltijoe — CHANGES_REQUESTED** [review]
Please atomize this

**Comment by Sniblet** [comment]
Frick that’s gonna be a lot of feature-frozen PRs
Give me a bit

**Comment by Moltijoe** [comment]
> Frick that’s gonna be a lot of feature-frozen PRs Give me a bit

just the spell checks from the buffs, not each one

**Review by Moltijoe — APPROVED** [review]
everything checks out

**Pull request merged**